AMICI requires sympy > 1.6

**To Reproduce**
```python
import amici
```

Produces
```
    import amici
/usr/local/lib/python3.7/site-packages/amici/__init__.py:134: in <module>
    from .sbml_import import SbmlImporter, assignmentRules2observables
/usr/local/lib/python3.7/site-packages/amici/sbml_import.py:26: in <module>
    from .ode_export import (
/usr/local/lib/python3.7/site-packages/amici/ode_export.py:35: in <module>
    from sympy.printing.cxx import _CXXCodePrinterBase
E   ModuleNotFoundError: No module named 'sympy.printing.cxx'
```

**AMICI version and system environment**
 - OS and version: [e.g. Ubuntu, iOS, Windows]: Ubuntu
 - AMICI interface: [e.g. Python, Matlab, C++]: Python
 - AMICI version: 0.11.11
 - Additional information: Compiler name and version used, Python/Matlab version, ...

**How to fix**
This is fixed by installing a more recent version of sympy (e.g., 1.7.1). I suspect the minimum required version is 1.7

`setup.py` needs to be updated to reflect the new requirement.

--- a/python/sdist/setup.py
+++ b/python/sdist/setup.py
@@ -200,7 +200,7 @@ def main():
                 'amici_import_petab.py = amici.petab_import:main'
             ]
         },
-        install_requires=['sympy>=1.6.0',
+        install_requires=['sympy>=1.7.1',
                           'python-libsbml',
                           'h5py',
                           'pandas',

